src/components/CurriculumComponents/helpers/curriculumMetadata.ts

```diff
@@ -19,31 +19,40 @@ function getExamboardFromSlug(props: buildCurriculumMetadataProps) {
 export const buildCurriculumMetadata = (
   props: buildCurriculumMetadataProps,
 ) => {
+  const { keyStages } = props;
+  const keyStagesItems: string[] = [];
+  if (keyStages.includes("ks1")) keyStagesItems.push("1");
+  if (keyStages.includes("ks2")) keyStagesItems.push("2");
+  if (keyStages.includes("ks3")) keyStagesItems.push("3");
+  if (keyStages.includes("ks4")) keyStagesItems.push("4");
+  const keyStagesData =
+    keyStagesItems.length > 0 ? `K${keyStagesItems.join("-")}` : ``;
```

Comment:
It probably doesn't matter if this is just a spike branch, but just to note that this should be:
```suggestion
    keyStagesItems.length > 0 ? `KS${keyStagesItems.join("-")}` : ``;
```